enterprise/cmd/executor/internal/worker/workspace.go

```diff
@@ -28,19 +30,24 @@ func (h *handler) prepareWorkspace(ctx context.Context, commandRunner command.Ru
 	}()
 
 	if repositoryName != "" {
-		cloneURL, err := makeURL(
+		cloneURL, err := makeRelativeURL(
 			h.options.ClientOptions.EndpointOptions.URL,
-			h.options.ClientOptions.EndpointOptions.Password,
 			h.options.GitServicePath,
 			repositoryName,
 		)
 		if err != nil {
 			return "", err
 		}
 
+		authorizationOption := fmt.Sprintf(
+			"http.extraheader='Authorization: %s %s'",
```

Comment:
```suggestion
			"http.extraHeader='Authorization: %s %s'",
```

Not sure whether git is case sensitive here, but why not?